### PR TITLE
SS-59 page names in title

### PR DESCRIPTION
**Pull request by churnikov**
## Status

- [ x ] Ready
- [ ] Draft
- [ ] Hold

## Description

I added custom titles to all pages and tried to add tests for them. It's not really trival as it turns out.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [ x ] This pull request is against **develop** branch (not applicable for hotfixes)
- [ x ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: requirements.txt
Comment (akochari) [inline requirements.txt:1]: Why do we need these?

**Review by akochari — APPROVED** [review]
Looks good to me!

File: cypress/e2e/ui-tests/test-public-apps-page.cy.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ describe("Test of the public apps page", () => {
     it("should contain header with text Apps", () => {
 
         cy.get('h3').should('contain', 'Apps')
+        cy.get("title").should("have.text", "Overview | SciLifeLab Serve")
```
Comment (alfredeen) [inline cypress/e2e/ui-tests/test-public-apps-page.cy.js:11]: Should the apps view really have title "Overview | SciLifeLab Serve"?

File: requirements.txt [hunk 2]
```diff
@@ -49,4 +49,9 @@ uvicorn==0.20.0
 
 # Pytest Django
 pytest-django==4.5.2
-pytest-cov==4.1.0
\ No newline at end of file
+pytest-cov==4.1.0
+
+# Mocking
+moto[ec2,s3,all]==4.2.2
```
Comment (alfredeen) [inline requirements.txt:55]: Nice that you use moto :-)

File: cypress/e2e/ui-tests/test-public-apps-page.cy.js (hunk 1 above)
Comment (churnikov) [reply cypress/e2e/ui-tests/test-public-apps-page.cy.js:11]: I think that's the question to @akochari 
I think `Apps` would have been fine

File: requirements.txt
Comment (churnikov) [reply requirements.txt:1]: For tests and mocking s3 connection

File: cypress/e2e/ui-tests/test-login.cy.js [hunk 3]
```diff
@@ -24,6 +24,9 @@ describe("Test login", () => {
     it("can login an existing user through the UI when input is valid", () => {
 
         cy.visit("accounts/login/")
```
Comment (sandstromviktor) [inline cypress/e2e/ui-tests/test-login.cy.js:26]: So this fails...

File: cypress/e2e/ui-tests/test-login.cy.js [hunk 4]
```diff
@@ -41,4 +44,9 @@ describe("Test login", () => {
         cy.loginViaUI(users.login.username, users.login.password)
 
     })
+
+    it("Should have proper title", () => {
+	cy.visit("accounts/login/")
```
Comment (sandstromviktor) [inline cypress/e2e/ui-tests/test-login.cy.js:49]: but not this... I don't understand.

**Review by sandstromviktor — APPROVED** [review]
Great job! I like the new changes!

File: cypress/e2e/ui-tests/test-public-apps-page.cy.js (hunk 1 above)
Comment (churnikov) [reply cypress/e2e/ui-tests/test-public-apps-page.cy.js:11]: @akochari What do you think?

Comment (akochari) [reply cypress/e2e/ui-tests/test-public-apps-page.cy.js:11]: Yes, agree, should be Apps.

**Pull request merged**